SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -612,50 +612,37 @@ void CopyWellKnownSchemaFiles(const FString& GDKSchemaCopyDir, const FString& Co
 	
 	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
 
-	if (!PlatformFile.DirectoryExists(*GDKSchemaCopyDir))
-	{
-		if (!PlatformFile.CreateDirectoryTree(*GDKSchemaCopyDir))
-		{
-			UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Could not create gdk schema directory '%s'! Please make sure the parent directory is writeable."), *GDKSchemaCopyDir);
-		}
-	}
-
-	if (!PlatformFile.DirectoryExists(*CoreSDKSchemaCopyDir))
-	{
-		if (!PlatformFile.CreateDirectoryTree(*CoreSDKSchemaCopyDir))
-		{
-			UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Could not create standard library schema directory '%s'! Please make sure the parent directory is writeable."), *GDKSchemaCopyDir);
-		}
-	}
-
+	RefreshSchemaFiles(*GDKSchemaCopyDir);
 	if (!PlatformFile.CopyDirectoryTree(*GDKSchemaCopyDir, *GDKSchemaDir, true /*bOverwriteExisting*/))
 	{
 		UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Could not copy gdk schema to '%s'! Please make sure the directory is writeable."), *GDKSchemaCopyDir);
 	}
 
+	RefreshSchemaFiles(*CoreSDKSchemaCopyDir);
```

Comment:
This is called twice, does it need only calling only once either before or after the above CopyDirectoryTree?